test/FirewallTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * Firewal tests
+ *
+ * @package Test
```

Comment:
please remove package and add a file-level docBlock with at least @copyright, @license